src/globalComponents/apiCalls/marketPlaceAPIConnections.js

```diff
@@ -68,7 +68,8 @@ function getMarketSales(userId: string, onLoad:(sales: Array<SaleObject>) => voi
 						sale.salePrice,
 						sale.amount,
 						sale.sellerId,
-						sale._id
+						sale._id,
+						"0"
```

Comment:
In JS single quotes are preferred according to our style guide.

Also, this should probably be an empty string

Reply:
On second thought, it should definitely be null, not empty string. I didn't fully read what this was being used for.